lib/app_registry.js

```diff
@@ -44,4 +44,15 @@ class Home extends React.Component {
 }
 
 AppRegistry.registerComponent('Artist', () => Artist);
-AppRegistry.registerComponent('Home', () => Home);
\ No newline at end of file
+AppRegistry.registerComponent('Home', () => Home);
+
+// How can I make this only happen in debug mode?
```

Comment:
Not sure if that will work with `import`, though, you might need to switch to using `require`.